src/triggered/Merge.ts

```diff
@@ -0,0 +1,47 @@
+import { css, html, LitElement, query, TemplateResult } from 'lit-element';
+
+import { newWizardEvent } from '../foundation.js';
+import { mergeWizard } from '../wizards.js';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7195001)
```suggestion
import { mergeWizard } from "../wizards.js";
```